sopel/tools/target.py

```diff
@@ -50,16 +50,23 @@ def __init__(
         This maps channel name :class:`~sopel.tools.identifiers.Identifier`\\s
         to :class:`Channel` objects.
         """
-        self.account = None
+        self.account: Optional[str] = None
         """The IRC services account of the user.
 
         This relies on IRCv3 account tracking being enabled.
         """
-        self.away = None
+        self.away: Optional[bool] = None
         """Whether the user is marked as away."""
```

Comment:
I bet this docstring could be improved the same way it is for `is_bot`, by telling why `None` is an acceptable value here.

Reply:
Bunch of docstring tweaks along these lines coming, not only for `is_bot`. Great idea!